airflow/models/taskinstance.py

```diff
@@ -1670,6 +1696,9 @@ def get(
                 'json': VariableJsonAccessor(),
                 'value': VariableAccessor(),
             },
+            'conn': {
+                'value': ConnectionAccessor(),
+            },
```

Comment:
```suggestion
            'conn': ConnectionAccessor(),
```

We don't need the `value` subitem here, as conn can only ever return one thing.

Reply:
But don't we want consistency between the var accessor syntax and the connection accessor syntax? I mean, assuming people is already familiar with  syntax `var.value.my_connection` I guess they will try `conn.value.my_connection` and not `conn.my_connection`. 

If you already pondered about that and still think the `conn.my_connectio` is preferrable just tell me and I'll change it.

Reply:
So for variable is makes sense as there are two different ways of accessing a variable (as a plain string, or trying to json parse it) but I can't think of any case where we'd need this for connections, which always return "structured" data.

Have I missed something?

Reply:
No , there is **no need**, it's just for having a **consistent api for both var and conns**. I mean not having to remember that when you access a variable you  need `var.value.xxx` but when you access a connection is just `conn.xxx`. But sure, I'll change it.